src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -159,8 +159,17 @@ public bool IsConfidentialClient
 #endif
             }
         }
+
+        /// <remarks>
+        /// Is null when <see cref="ILongRunningWebApi.AcquireTokenInLongRunningProcess"/> is called.
```

Comment:
is it in the public API?

Reply:
No, internal. Just added a comment for our clarification.